compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -854,9 +838,6 @@ static LogicalResult setMatmulNoPadRootConfig(
   const SmallVectorImpl<bool> &vecScalableDims = inputScalableTileFlags.back();
   SmallVector<int64_t> parallelTileSizes;
   SmallVector<bool> parallelScalableFlags;
-  bool allowIncompleteTile =
-      vecPreProcStrategy == VectorPreProcStrategy::Peeling ||
-      vecPreProcStrategy == VectorPreProcStrategy::Masking;
```

Comment:
Do we need to assert if the strategy is not one of these?

Reply:
I think it is okay because all the matmul strategy is using peeling or masking. At least it is the case for targets we've been actively working on.